cmd/deploy/deploy.go

```diff
@@ -511,9 +511,34 @@ func getDefaultTimeout() time.Duration {
 	return parsed
 }
 
-func GetDeployer(ctx context.Context, manifest *model.Manifest, opts *Options, builder *buildv2.OktetoBuilder, cmapHandler configMapHandler) (deployerInterface, error) {
+func shouldRunInRemote(opts *Options) bool {
```

Comment:
we need to add unit tests to this function, even more considering its importance. The same for the function `shouldRunInRemote` on the destroy command although they are practically similar.